docker/dev/docker-compose.yml

```diff
@@ -48,7 +48,6 @@ services:
       FROM_EMAIL: ${FROM_EMAIL}
       SEND_EMAIL: ${SEND_EMAIL:-false}
       REDIS_URL: redis://redis:6379/0
-      KRATOS_ADMIN_URL: 'http://kratos:4434/admin'
```

Comment:
lets keep it, and just introduce the `AUTH_ENABLED` flag with a default `false` value
people may still want to try kratos locally as well